pkg/units/control.go

```diff
@@ -94,48 +95,121 @@ func (cu *Control) RunFuzzer() error {
 			continue
 		}
 
-		encodedProg, err := ebpf.EncodeInstructions(prog)
-		if err != nil {
-			fmt.Printf("Encoding error: %v\n", err)
-			if !cu.strat.OnError(err) {
-				return err
+		switch p := prog.Program.(type) {
+		case *pb.Program_Cbpf:
+			err := cu.runCbpfFuzzer(p.Cbpf)
```

Comment:
could we name the functions just `runCbpf` and `runEbpf`?

Reply:
Done